### ISSUE TITLE
Crash when adding server using touchscreen

### ISSUE THREAD

**Issue by LAGonauta** [rymate1234/WinIRC#2]
Hello, for some reason the app crashes when I try to add a server on my ThinkPad 8 using the touchscreen. Luckily I also had my bluetooth keyboard connected and was able to add the server using TAB.

I am also unable to connect to any server, but this that is for another bug report (or I just need to learn how to use WinIRC correctly).

**Comment by jmurth1234** [comment]
Hi, thanks for reporting the issue.

Any chance you could give some details as to what settings are causing the issues? I know on my devices I am able to save servers via the touchscreen, so any details that will allow me to reproduce the issue would be extremely helpful.

**Comment by LAGonauta** [comment]
Sure. I do not know what kind of setting could influence this, but I can say that my screen is at 175%, it is 1920x1200 and the tablet specs are:

Lenovo ThinkPad 8
Windows 10 Anniversary Edition 32-bit
Atom Z3770, 2GB RAM

I tried adding a server on a recently installed WinIRC with all defaults. It crashes as soon as I press on any cell on the form to write content, but the toggles work fine.

**Comment by jmurth1234** [comment]
Wait, so this error isn't when hitting the save button, just when touching one of the entry fields?

That's.... an extremely strange issue.

This 100% didn't happen on my lumia, however I've just managed to reproduce on my tablet, which is strange as I'm sure it didn't happen before o.o

A fix will be included in the 1.4 release of WinIRC, which will be released once I've fixed a couple issues with some of the new behaviour I've added.
